### PR TITLE
Docs(web-react, web): Add Heading demo #DS-899

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

Heading demo in web-react and web is now same as demo in web-twig

### Additional context

ℹ️ Spaces are now different, we need fix CSS of `DocsStack` in all packages (WIP)

### Issue reference

[Demo: Heading má ve všech technologiích stejná dema](https://jira.lmc.cz/browse/DS-899)

### CODE REVIEW

File: packages/web/src/scss/components/Heading/README.md
Comment (literat) [inline packages/web/src/scss/components/Heading/README.md:1]: ❤️

Comment (crishpeen) [reply packages/web/src/scss/components/Heading/README.md:1]: I don't think so. There isn't actually a Heading component. It is only a helper. My suggestions are to either showcase this in the helpers section or drop all changes in this package.

**Review by crishpeen — COMMENTED** [review]
please remove web packages changes

**Comment by pavelklibani** [comment]
> please remove web package changes

Removed

File: packages/web/src/scss/components/Heading/README.md
Comment (pavelklibani) [reply packages/web/src/scss/components/Heading/README.md:1]: Heading component was removed from web components, we have its demo in web helpers.

**Review by crishpeen — APPROVED** [review]

**Pull request merged**